### ISSUE TITLE
bug in log file

### ISSUE THREAD

**Issue by p060477** [DNSCrypt/dnscrypt-proxy#2304]
THE TRACKER IS DEDICATED TO KEEPING TRACK OF *BUGS*,
preferably after they have been already discussed and confirmed to be reproducible.

FOR ASSISTANCE, PLEASE CLOSE THIS FORM AND USE THE DISCUSSIONS SECTION INSTEAD:
https://github.com/DNSCrypt/dnscrypt-proxy/discussions/categories/q-a

~~~

Reported bugs must reproducible in the context described in the "Context" section.

Installation and configuration issues are not bugs, but individual assistance request.

Context: the LATEST version of `dnscrypt-proxy` (precompiled binaries downloaded from this repository) is correctly installed and configured on your system, but something doesn't seem to produce the expected result.

If the bug is not trivial to reproduce on any platform, please include ALL the steps required to reliably duplicate it, on a vanilla, generic install of macOS, Windows, OpenBSD or Ubuntu Linux system, in their most current version.

If you don't have any clear understanding of the issue or can't enumerate the steps to reproduce it, open a discussion instead:
https://github.com/DNSCrypt/dnscrypt-proxy/discussions


## Output of the following commands:

./dnscrypt-proxy -version

./dnscrypt-proxy -check

./dnscrypt-proxy -resolve example.com


- [ ] Initially raised as discussion #...



## *What* is affected by this bug?



## *When* does this occur?



## *Where* does it happen?



## *How* do we replicate the issue?
<!-- Please list all the steps required to reliably replicate it, starting from a newly installed operating system -->



## Expected behavior (i.e. solution)



## Other Comments

again another mistake by you:

`N:\DESKTOP\PC\Dns\dnscrypt-proxy>dnscrypt-proxy.exe -config dnscrypt-proxy.toml -check > dnscrypt-proxy.log

N:\DESKTOP\PC\Dns\dnscrypt-proxy>dnscrypt-proxy.exe -config dnscrypt-proxy.toml -check | tee dnscrypt-proxy.log
"tee" non è riconosciuto come comando interno o esterno,
un programma eseguibile o un file batch.
`

to sum up the first command is right:
dnscrypt-proxy.exe -config dnscrypt-proxy.toml -check > dnscrypt-proxy.log

but this:
dnscrypt-proxy.exe -config dnscrypt-proxy.toml -check | tee dnscrypt-proxy.log
is a -fault- :

N:\DESKTOP\PC\Dns\dnscrypt-proxy>dnscrypt-proxy.exe -config dnscrypt-proxy.toml -check | tee dnscrypt-proxy.log
"tee" non è riconosciuto come comando interno o esterno,
un programma eseguibile o un file batch.

so i re-re pray you again and again: be -precise-
thxs in adv
cheers
